pkg/cmd/application/create/app_create.go

```diff
@@ -121,11 +129,19 @@ NOTE:
 }
 
 func run(opts *AppCreateOpt, cfg *config.Config) error {
-	if _, err := config.ResetConfigRepositoryState(&cfg.Repositories.CPlatform); err != nil {
-		return err
+	defer opts.Streams.CurrentHandler.Done()
+
+	opts.Streams.CurrentHandler.Info(fmt.Sprintf("resetting local repository [repo=%s]", cfg.Repositories.CPlatform.Value))
+	if !cfg.DryRun {
+		if _, err := config.ResetConfigRepositoryState(&cfg.Repositories.CPlatform, cfg.DryRun); err != nil {
```

Comment:
Can we add more meaningfull error outputs? Like

return fmt.Errorf("failed to reset config repository state for CPlatform: %w", err)

Reply:
Added